site/content/en/docs/Integration Patterns/canary-testing.md

```diff
@@ -32,15 +32,13 @@ To ensure we don't have to change the Allocation system every time we have a can
 state that in our system, the label `canary: "true"` will be added to any canary `Fleet` in the cluster.   
 
 ```yaml
-apiVersion: "allocation.agones.dev/v1"
+apiVersion: allocation.agones.dev/v1
 kind: GameServerAllocation
 spec:
-  preferred:
+  selectors:
     - matchLabels:
         canary: "true"
-  required:
-    matchLabels:
-      agones.dev/fleet: stable
+        agones.dev/fleet: stable
```

Comment:
So right now this is a "canary and fleet is 'stable'" - what we actually want is "check the canary label'ed GameServers, and if there are none left, then go to the 'stable' fleet.

So we want this instead:

```yaml
apiVersion: allocation.agones.dev/v1
kind: GameServerAllocation
spec:
  selectors:
    - matchLabels:
        canary: "true"
    - matchLabels:
        agones.dev/fleet: stable
```

Which will first look for the first selector, and if it fails go to the second.

Reply:
I've made the changes you recommended.